.gitignore

```diff
@@ -15,6 +15,7 @@ eshell/lastdir
 my-org/
 org-files/
 semanticdb/
+srecode-map.el
```

Comment:
btw, I have nothing beyond the standard c-c++ layers so don't think it (should) be anything particular to my setup.